data/mods/Xedra_Evolved/enchantments/mutations.json

```diff
@@ -25,5 +25,21 @@
     "id": "BONUS_INVENTOR",
     "condition": "ALWAYS",
     "values": [ { "value": "MAX_MANA", "add": { "math": [ "u_skill('deduction')*50" ] } } ]
+  },
+  {
+    "type": "enchantment",
+    "id": "hunter_of_weird",
+    "condition": "ALWAYS",
+    "hit_you_effect": [ { "id": "eater_sixth_sense", "hit_self": true, "once_in": 12 } ]
+  },
+  {
+    "id": "eater_sixth_sense",
+    "type": "SPELL",
+    "name": "Eater Sixth Sense",
+    "description": "Heals the caster.",
+    "valid_targets": [ "self" ],
+    "effect": "sense_unnatural",
```

Comment:
> "effect": "sense_unnatural",
"effect": "attack",
"effect_str": "sense_unnatural",

Needs a duration as well.

Reply:
I don't know how to do the fancy Github changes form so I just used quotes. 🥴